lit_gpt/lora.py

```diff
@@ -143,10 +144,11 @@ def merge(self):
     def forward(self, x: torch.Tensor):
         # if weights are merged or rank is less or equal to zero (LoRA is disabled) - it's only a regular nn.Linear forward pass;
         # otherwise in addition do the forward pass with LoRA weights and add it's output to the output from pretrained weights
-        result = self.linear(x)
-        if self.r > 0 and not self.merged:
-            result += (self.lora_dropout(x) @ self.lora_A.transpose(0, 1) @ self.lora_B.transpose(0, 1)) * self.scaling
```

Comment:
A question: 
```python
result = result + ...
```
didn't do the trick?

Or you just decided to go even further: use proper names (instead of result) to better name each stage?